patches/lua-tree.patch

```diff
@@ -415,14 +410,8 @@ index 6c92098cc..c3c3c12fe 100644
  local type          = type
  local pairs         = pairs
  local ipairs        = ipairs
-@@ -33,34 +26,12 @@ local fmt           = string.format
- local find          = string.find
- local gsub          = string.gsub
- local split         = pl_stringx.split
--local re_find       = ngx.re.find
--local re_match      = ngx.re.match
-+local re_find       = require "go.re2".find
-+local re_match      = require "go.re2".match
+@@ -30,30 +30,8 @@ local fmt           = string.format
+ local re_match      = ngx.re.match
```

Comment:
Since I'm also [removing pre-loading the gluare package](https://github.com/Kong/goks/pull/26/files#diff-d1124e8991e10b74a2af9007919a02ed1c63287e2d8e82134fe0388f89934ebcL58), I've removed `re_find` where it is being declared but it is unused. I've only done so where we were previously using `"go.re2".find`. I would probably need to keep the pre-loader in `vm.go` to accurately leave this unchanged.